example/pointsocial.z/README.md

```diff
@@ -20,22 +20,17 @@ Now a `public` folder will be created containing the deployable built site. You
 
         cd example/pointsocial.z
         npm i
-        npm run watch
-
-1. Now, **stop** the `watch` command that is running from the previous step (use `CTRL+C` to stop the `watch` process).
-1. Delete the generated `index.js` and `index.js.map` files from the `example/pointsocial.z/public` folder and create empty placeholder files here instead.
-
-        rm public/index.js
-        rm public/index.js.map
-        touch public/index.js
-        touch public/index.js.map
-
-1. Comment out [lines 151](https://github.com/pointnetwork/pointnetwork/blob/master/db/models/file.js#L151) & 152 in `db/models/file.js` class. (NOTE: **do not** commit this change to your branch! Always revert this change before making the commit.).
-1. (OPTIONAL) Stop the nodes using docker: `docker-compose -f docker-compose.yaml -f docker-compose.dev.yaml down -v`
-1. Start the node using docker: `docker-compose -f docker-compose.yaml -f docker-compose.dev.yaml up -d`
+        npm run build
+
+1. Update `.env.e2e` with your local account Private Key and Initial Balance in the `ACCOUNT_PK_AND_BALANCE` env key.
+1. Update `.env.e2e` with your the Zapp domain that you are developing in the `DEV_ZAPP_HOST` env key (i.e. set this to 'pointsocial.z').
+1. In `StorageProviderRegistry.sol` update the `connection` of the defualt provider in the constructor function to: http://storage_provider:9685/#c01011611e3501c6b3f6dc4b6d3fe644d21ab301 (**TODO**: need to avoid having to do this)
+1. Stop the nodes using docker: `docker-compose --env-file .env.e2e -f docker-compose.e2e.yaml -f docker-compose.dev.yaml down -v`
```

Comment:
And improvement suggestion. Later we could add a handy npm script for that, because these commands are too easy to make a typo in. But if we create some script say `"e2e:compose": "docker-compose --env-file .env.e2e -f docker-compose.e2e.yaml -f docker-compose.dev.yaml"`, a developer will only have to type `npm run e2e:compose -- up -d` and `npm run e2e:compose -- down` which is much simpler.